src/requirements.txt

```diff
@@ -1,9 +1,44 @@
+argh==0.26.2
```

Comment:
Wow this is a lot of new requirements! Where did they all come from?

Reply:
I was installing the mistune contrib module and trying to update the the requirements but didn't realized it somehow generated all of these :|